docs/index.rst

```diff
@@ -175,7 +175,7 @@ Maintaining the system
 
 * :doc:`/user_guide/mh_mechanical_maintenance`
 * :doc:`/user_guide/mh_electronic_maintenance`
-* :doc:`/user_guide/mh_installing_the_software`
+* :doc:`/user_guide/mh_re_installing_the_software`
```

Comment:
I dont think you need to change that here, as you haven't renamed the file. Inside it you can change the title as you have done.